utils/utils.go

```diff
@@ -25,3 +26,9 @@ func CleanupDirectory(fn string) {
 		log.Debugf("\tCleaned up: %s", fn)
 	}
 }
+
+
+func OsGnostic_HandleAwsKey(org string, folder string, fn string) string {
```

Comment:
should this be "Agnostic"? or am i not getting a pun

Reply:
for sure - this was my thinking:

on linux/macs the .ToSlash is redundant, and wont modify the string, it only matters on Windows.

so it may be better programming if I explicitly identify the platform to determine if I need to make this modification. Yes, as of now it is agnostic.

whats your opinion, explicitly identify the platform or change the name?

Reply:
FYI - I did the lesser work here, just changing to agnostic. we will go down the other route depending on our needs.